static/js/server_events_dispatch.js

```diff
@@ -22,6 +22,12 @@ exports.dispatch_normal_event = function dispatch_normal_event(event) {
             event.hotspots;
         break;
 
+    case 'invites_changed':
+        if ($('#admin-invites-list').length) {
```

Comment:
why this check for non-zero length? 
I guess currently we don't send_event() for when invites are added, but we should just send an event for that as well :).

Reply:
This check is there to see whether or not the invitations tab is open because otherwise an error will be thrown if we call settings_invite.set_up().
The event is being sent when invites are added.